include/odp/api/spec/classification.h

```diff
@@ -178,6 +179,60 @@ typedef struct odp_bp_param_t {
 	odp_threshold_t threshold;
 } odp_bp_param_t;
 
+/**
+ * Classifier queue specific statistics counters
+ */
+typedef struct odp_cls_queue_stats_t {
+	/** Number of octets in packets delivered to the queue. In case of
+	 *  Ethernet, packet size includes MAC header and FCS. */
```

Comment:
@MatiasElo Hi, This is Sunil Kumar Kori from Marvell. Sorry for commenting from my personal account as my official account is not working for some reason. 

As per spec, octets contains FCS also. So I just wanted to understand the use case of adding it. Because in general cases FCS are striped  at H/W and remaining packet size is updated as a stats.

Reply:
Hi @sunilkori, I added this text to be compatible with the original `odp_pktio_stats_t.in_octets` spec. If this proves to be problematic we'll have to also modify the old API as these counter definitions should match.

An easy implementation fix would  be to just add  `odp_cls_queue_stats_t.packets * 4` to the value returned by HW.

Reply:
That I understand and easily doable . My only point is to understand the usage of this. But if there is no specific use case then I believe, changing ODP spec is better.

Reply:
Fixed in V4